README.md

```diff
@@ -21,10 +21,131 @@ It implements the following architecture:
+      - Add the following values to the given variables of the Rapid Environment Editor:
+      
+                `Path`: (Expandable string)
+                    '<path to codes workspace>\glew\glew-2.1.0\glew-2.1.0\bin\Release\x64'
+                    '<path to codes workspace>\glew\glew-2.1.0\glew-2.1.0\bin\Debug\x64'
+  
+  
+      - Don't forget to configure, generate the cmake (cmake gui --> robotology yarp) and build release mode the yarp using vs15.  
+      
+      - if you have problems to install [git repo](https://github.com/nigels-com/glew) may help you. 
+     
+     
+     
+   3. `GLFW3`: To download this library go to [website1](https://www.glfw.org/) or [website2](https://www.glfw.org/download.html). Download the "Source package" among the available ones, the version of it is "3.2.1".
+     
+
+      - Place the package in your workspace (in our case in same path of robotology-superbuild)
+          
+
+      - Use CMake GUI to to configure and generate the project.
+          
+
+      - After that using the VS15 (VS 2017) build the project.
+          
+
+      - Add the following paths to the variables of robotology/YARP (because the yarpdev which gets the data from the sdk of the ovr and publishes in yarp framework, has dependencies on this library)
+          
+              `GLFW3_DIR`: '<path to codes workspace>/glfw/glfw-3.2.1/glfw-3.2.1/build/install/lib/cmake/glfw3' (inside this folder you should find the cmake files)
+          
+              `GLFW3_INCLUDE_DIR`: '<path to codes workspace>/glfw/glfw-3.2.1/glfw-3.2.1/build/install/include' (inside that you should find the GLFW folder, and inside it the header files)
+          
+              `GLFW3_OPENGL_DIR`: '<path to codes workspace>/glfw/glfw-3.2.1/glfw-3.2.1/build/install/lib' (you should be able to find the glfw.lib file)
+              `YARP_USEGLFW3`: check the box
+
+      - Add the following varibales to the Rapid Evironment Edittor:
+             
+              `GLFW3_DIR`= '<path to codes workspace>\glfw\glfw-3.2.1\glfw-3.2.1\build\install' (variable type is String)
+             
+      - Don't forget to configure, generate the cmake (cmake gui --> robotology yarp) and build release mode the yarp using vs15.  
+             
+      - if you have problems with compiling GLFW, [this link](https://www.glfw.org/docs/latest/compile_guide.html#compile_generate) or [here](https://github.com/nigels-com/glew) may help.
+          
+          
+    4. `LibOVR`: to dowload the SDK, you can go to [this website](), choose "Native Windows", then "Core Package: OCULUS SDK for windows". select the version 1.16.0 to download (you can install the latest one as well 1.32 (the last release) but there are some breaking changes in the APIs in version 1.17 and you will probably need to patch the driver software). To download you can go easily to [link](https://developer.oculus.com/downloads/package/oculus-sdk-for-windows/1.16.0/) to download the sdk version 1.16.0.
```

Comment:
`LibOVR: to dowload the SDK, you can go to [this website]()`. Link missing. + Typo (dowload).

Since the version 1.32 is not compatible with YARP you can directly write: 

4. `LibOVR`: to download the SDK, you can go to [this website](), choose "Native Windows", then "Core Package: OCULUS SDK for windows" and select the version 1.16.0. Or easily follow this [link](https://developer.oculus.com/downloads/package/oculus-sdk-for-windows/1.16.0/).